build.gradle

```diff
@@ -25,6 +25,7 @@ javadoc {
 }
 
 repositories {
+	mavenLocal()
```

Comment:
Max, I added this so I could use locally installed SNAPSHOT versions of devkitman. We can leave this in, right?

Reply:
hmm - I would say if it must be here it should be after mavenCentral() so  the build isn't injected with any local bits you have.